### PR TITLE
Power up info

### PR DESCRIPTION
**Pull request by guilbill**
## Description

Add some accessibility, and tests

## ToDo list:

- [x] on ne sait pas combien de powerUp il nous reste
- [x] on ne sait pas à quoi ils correspondent, leur effet
- [x] il manque de l'auto focus un peu partout
- [ ] support du clavier sur desktop

### CODE REVIEW

File: lib/game/Waiting.jsx [hunk 1]
```diff
@@ -20,7 +20,7 @@ const Waiting = ({ playerId, host, game }) => {
             <Text style={styles.title}>Tipsy</Text>
             <Text style={styles.text}>Waiting for opponent</Text>
             <Text style={styles.text}>Invite link :</Text>
-            <Text style={styles.text}>
+            <Text style={styles.text} accessibilityLabel="Join link">
```
Comment (djhi) [inline lib/game/Waiting.jsx:23]: I'd rather add an id on the Text above this one and an `accessibilityLabelledBy="idAbove"` on this one

File: lib/home/Welcome.jsx [hunk 2]
```diff
@@ -29,7 +29,7 @@ const Welcome = ({ playerName }) => {
             <Text style={styles.title}>Tipsy</Text>
             <Text style={styles.text}>Welcome {playerName}</Text>
             <AdaptiveButton
-                onPress={handleNewGame}
+                onPress={() => handleNewGame(false)}
```
Comment (djhi) [inline lib/home/Welcome.jsx:32]: If `quickGame` is `false` by default in `handleNewGame`, you don't need to pass it right? Not important though

**Review by djhi — CHANGES_REQUESTED** [review]

File: lib/home/Welcome.jsx (hunk 2 above)
Comment (guilbill) [reply lib/home/Welcome.jsx:32]: It didn't work when I did it... Don't understand why, and no time to investigate :)

File: lib/game/Waiting.jsx (hunk 1 above)
Comment (guilbill) [reply lib/game/Waiting.jsx:23]: If I put id on this Text component (react-native-web),  I don't see it in the resulting html...

Comment (djhi) [reply lib/game/Waiting.jsx:23]: see https://necolas.github.io/react-native-web/docs/text/, you have to use `nativeID`

**Review by djhi — APPROVED** [review]

**Pull request merged**